scripts/Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM ubuntu:20.04
+FROM ubuntu:22.04
 
 ENV DEBIAN_FRONTEND noninteractive
 
 COPY . .
 
 RUN apt update && apt dist-upgrade -y
-RUN ./configure.sh
```

Comment:
The configure file was removed in a recent commit; therefore we no longer need to reference it.

Reply:
I did a bit of spelunking through some of the old history of the repo (before release), and located where this Dockerfile came from. As best I can recover, it served as a method of getting a development environment up with the current source tree imported to make it easy to have a reproducible/consistent setup.

While not strictly necessary for us to keep, I'm not yet certain if some of our ancillary tooling (namely, the Test Controller) depends on it. For now, I'm opting to keep it in and updated so that the dev use-case remains practical; but I would not oppose its removal once we've managed to ensure nothing else would break.